Sources/GRPCReflectionService/Server/ReflectionService.swift

```diff
@@ -378,3 +396,32 @@ extension Google_Protobuf_FileDescriptorProto {
     return names
   }
 }
+
+extension Result<Reflection_ServerReflectionResponse.OneOf_MessageResponse, GRPCStatus> {
+  func recover() -> Result<Reflection_ServerReflectionResponse.OneOf_MessageResponse, Never> {
+    self.flatMapError { status in
+      let error = Reflection_ErrorResponse.with {
+        $0.errorCode = Int32(status.code.rawValue)
+        $0.errorMessage = status.message ?? ""
+      }
+      return .success(.errorResponse(error))
+    }
+  }
+  func makeResponse(
```

Comment:
nit: leave a blank line between functions

```suggestion
  }

  func makeResponse(
```